### ISSUE TITLE
Use of the [fem] and [male].

### ISSUE THREAD

**Issue by jimmubreen** [JMdictProject/JMdictIssues#27]
These two tags are being used for both JMdict and JMnedict. About 80 JMdict entries are tagged "male", i.e. "male term, language" and over 20,000 JMnedict entries have the "male" tag, where it means a male name.

The question is: Do we just keep doing it this way?  Or do we create a separate tag set for the two dictionaries? After all, the they are rather different things. Ideally, they should be separated, but it's not actually causing a problem.

**Comment by ReneMalenfant** [comment]
"masc" might be superior to "male" for JMdict?

Also, we might consider changing the definition slightly.  Currently, "male" is defined as: "male term, language, or name".  In the interest of inclusivity, etc., "traditionally male" might be more appropriate.

**Comment by JMdictProject** [comment]
Yes, having "masc" for JMdict and "male" for JMnedict would work. Similarly perhaps "femn" for JMdict and "fem" for JMnedict?
So:
masc - trad. male language
male - male name
femn - trad. female language (or "feminine language"?)
fem - female name

Will that work?

**Comment by robinjmdict** [comment]
I don't think there's a problem with using the same tags for for both JMdict and JMnedict as it's clear what "male" and "fem" refer to in each dictionary. But I also don't object to the proposed changes. I'd go with "trad. female language" for the description if "masc" is "trad. male language".

**Comment by Marcusjmdict** [comment]
I agree with the proposed change, from a technical perspective I think it's bad to re-use the same tags for different meanings.
